app/views/static_pages/landing.html.haml

```diff
@@ -42,20 +46,26 @@
               providing the percentage difference between commits and releases.
               In the near future, RubyBench will also monitor and track popular Ruby
               libraries such as Rails.
-          .col-md-6.col-md-pull-6
-            = image_tag('landing/landing-plan.png', alt: 'RubyBench Plan', class: 'landing-graphic-medium')
 
         .row.landing-section
-          .col-md-6
+          .col-md-6.col-md-push-6
+            = image_tag('landing/landing-hardware.png', alt: 'RubyBench Hardware', class: 'landing-graphic-medium')
+
+          .col-md-6.col-md-pull-6
```

Comment:
Nope, these classes are meant for reversing the order of rendering on mobile widths, exactly what you are trying to do here. On hindsight, I think the way you are doing now is better: images should be before the blurb. Oops.